README.md

```diff
@@ -1,20 +1,53 @@
 # elrrdatasync
 ELRR services which aid in the streaming (Kafka streams) of data from the staging database to the ELRR database.
 
-# Requirements
-For building and running the elrrdatasync you need:
-- JDK 1.8
-- Maven 3
+There are database and kafka dependencies, but there's a repo with a docker-compose that resolves them locally.
+[.yaml](https://github.com/US-ELRR/elrrdatasync/tree/dev-pom-update-local-fixes-and-docs-1/dev-resources/docker-compose)
+
+Setup elrrdatasync first [README](https://github.com/US-ELRR/elrrdatasync/blob/dev-pom-update-local-fixes-and-docs-1/README.md)
+
+- Java JDK 1.8 [JDK](https://www.oracle.com/java/technologies/downloads/)
```

Comment:
Same thing here, like this is how it renders:

![Screenshot from 2024-10-29 11-04-09](https://github.com/user-attachments/assets/bb7c4575-9ae1-4a0a-be94-05c298bec975)

I assumed this was not intentional. Just linking the word itself is probably best.